### PR TITLE
plate armor clanking

### PR DESCRIPTION
**Pull request by grungussuss**
## About The Pull Request

https://github.com/user-attachments/assets/0c667403-89d1-4e55-b8e7-af91aca16779
## Why It's Good For The Game

immersion

## Changelog
:cl: grungussuss
sound: plate armor now makes sound when moving around in it
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
add to zerker and medieval pirate leader

**Comment by K4rlox** [comment]
add to chaplain templar armor?

**Comment by Sadboysuss** [comment]
> add to chaplain templar armor?

yea I did

**Comment by Admiral-Gigaton** [comment]
miner doom slayer armor (?) and adamantine armor, too!

File: code/modules/research/xenobiology/crossbreeding/_clothing.dm [hunk 1]
```diff
@@ -44,6 +44,7 @@ Slimecrossing Armor
 /obj/item/clothing/glasses/prism_glasses/Initialize(mapload)
 	. = ..()
 	AddElement(/datum/element/wearable_client_colour, /datum/client_colour/glass_colour, ITEM_SLOT_EYES, glasses_color, forced_glass_color)
+	AddComponent(/datum/component/item_equipped_movement_rustle, SFX_PLATE_ARMOR_RUSTLE, 8)
```
Comment (Ghommie) [inline code/modules/research/xenobiology/crossbreeding/_clothing.dm:47]: ♂ Hey buddy, I think you've got the wrong item, the adamantine armor is 100 lines down. ♂

Comment (Sadboysuss) [reply code/modules/research/xenobiology/crossbreeding/_clothing.dm:47]: Oops

**Pull request merged**